frontend/src/lib/components/PageHeader.tsx

```diff
@@ -7,7 +7,7 @@ import { DraggableToNotebookProps } from 'scenes/notebooks/AddToNotebook/Draggab
 import { breadcrumbsLogic } from '~/layout/navigation/Breadcrumbs/breadcrumbsLogic'
 
 interface PageHeaderProps {
-    title: string | JSX.Element
+    title?: string | JSX.Element
```

Comment:
not sure how this is being used at all 🤔 , is it now redundant, or does something I'm not sure about? @Twixes ?

Reply:
It seems redundant, as PageHeader doesn’t use it at all, and also, this interface is not exported.

There’s a bunch of PageHeader invocations where `title` is still being passed - happy to clean this up once @Twixes confirms.

Reply:
Yup, we can get rid of `title` now